modules/command/dc/dc-offer-create-db-command.js

```diff
@@ -71,6 +71,9 @@ class DCOfferCreateDbCommand extends Command {
         this.remoteControl.offerUpdate({
             id: internalOfferId,
         });
+        models.handler_ids.update({
+            status: 'FAILED',
```

Comment:
I'd recommend adding the error message to the handler data, or retrieving it from the offers table upon the API call. Otherwise, we'll only see failed, instead of seeing why it failed.